LICENSE.md

```diff
@@ -647,6 +647,28 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+### deck.gl
```

Comment:
The point occlusion shader uses 64-bit primitives internally if the USE_TRIANGLE flag is enabled